samples/resource.json

```diff
@@ -33,7 +33,10 @@
     "id": "ae3a5d82-cdd1-468d-9bc9-3aa9e54d953c"
   },
   "_dd": {
-    "format_version": 2
+    "format_version": 2,
+    "resource_id": "b83a5d82-cdd1-468d-9bc9-3aa9e54d957a",
```

Comment:
specifically for resource ID, shouldn't we be consistent with view, action, etc and have it be `@resource.id`?

Reply:
It should be indeed more consistent, fine by me.
@xgouchet?

Reply:
Yep 👍